### PR TITLE
New Chapter - Silicon Valley PyLadies

### PR DESCRIPTION
**Pull request by posenberg**
Hi there,

Sorry for the delay. I waited too long, so I re-forked the repo instead of squashing the commits. I've updated the config.yml, added a logo to the _assets folder, and added the siliconvalley folder.

Thank you so much!

### CODE REVIEW

File: www/config.yml [hunk 1]
```diff
@@ -551,6 +551,19 @@ chapters:
             latitude: 37.5666791
             longitude: 126.9782914
 
+    - name:      Silicon Valley, CA
+      meetup_id: 19180337
+      website: http://siliconvalley.pyladies.com
+      image: pyladies-silicion-logo.png
```
Comment (econchick) [inline www/config.yml]: This is incorrect - it should be `.jpeg` :)

**Comment by econchick** [comment]
hey @posenberg - just a quick edit, you put `.png` instead of `.jpeg` for your image, and therefore it doesn't render:

2016-01-11 at 12 39 pm

I won't merge this until you fix it, but I will deploy to siliconvalley.pyladies.com (should be up in a couple of minutes) for now.

**Comment by posenberg** [comment]
I fixed the image problem, but I'm having trouble squashing my commits. I either end up with an extra commit, or I have a few new commits in $ git log.

**Comment by econchick** [comment]
@posenberg sorry for the delay - merged and deployed.

Re: your issues with commit stuff - for future FYI, I think you actually needed to pull from master again, and rebase your branch against master. But no worries - I got it to work :) 

LMK if anything looks weird!

**Pull request merged**